framework/src/meshgenerators/XYDelaunayGenerator.C

```diff
@@ -138,7 +147,8 @@ XYDelaunayGenerator::XYDelaunayGenerator(const InputParameters & parameters)
     _auto_area_function_power(getParam<Real>("auto_area_function_power")),
     _algorithm(parameters.get<MooseEnum>("algorithm")),
     _tri_elem_type(parameters.get<MooseEnum>("tri_element_type")),
-    _verbose_stitching(parameters.get<bool>("verbose_stitching"))
+    _verbose_stitching(parameters.get<bool>("verbose_stitching")),
+    _interior_points(getParam<std::vector<Point>>("interior_points"))
```

Comment:
As we now have two potential ways to assign values for `_interior_points`, we should move it out of the initializer list.